src/lib/components/AimeVideoPlayer/AimeVideoPlayer.module.scss

```diff
@@ -4,15 +4,14 @@
 
 .playerContainer {
   width: 100%;
-  z-index: 20;
   height: 100%;
   display: flex;
+  z-index: 12000;
```

Comment:
😱

Reply:
hahaha, yes I know @lulen11 I want to make sure the AVP is on top of everything!!!
I don't know the accepted practice for z-index, I'm not sure if there is a general accepted approach? 
Design and css will need to be heavily reviewed or if its easier feel free to make the changes whichever is best 😃